nautobot/project-static/js/job_result.js

```diff
@@ -31,15 +31,7 @@ $(document).ready(function(){
                 context: this,
                 success: function(data) {
                     updatePendingStatusLabel(data.status);
-                    if (data.status.value === 'completed' || data.status.value === 'failed' || data.status.value === 'errored'){
-                        jobTerminatedAction()
-                    } else {
-                        setTimeout(checkPendingResult, timeout);
-                        if (timeout < 10000) {
-                            // back off each iteration, until we reach a 10s interval
-                            timeout += 1000
-                        }
-                    }
+                    setTimeout(jobTerminatedAction, 1000);
```

Comment:
This seems wrong in context. Why are we even bothering with an AJAX call to the REST API (above) any more, if we just automatically reload the page every second?

Reply:
Right yeah. It was more just to get it working. I'm trying to coerce `django-ajax-tables` in the interim.